gem/static/css/_style.surveys.scss

```diff
@@ -0,0 +1,137 @@
+// Chris Sergienko - Praekelt Consulting (2016/09/30)
+
+.surveys {
+    $block: unquote("#{&}"); // Node LibSass workaround for @at-root bug. This stores the value of `&` as a variable.
+    padding: 24px;
+
+    &__title {
+        line-height: 1.15;
+        font-family: $font-family-avenirBold;
+    }
+
+    &__intro {
+        font-family: $font-family-avenir-ot;
+    }
+
+    &__helptext {
+        display: block;
+        font-size: 16px;
+        font-size: 1rem;
+    }
+
+    &__form {
+        padding: 24px 16px;
+        padding: 1.5rem 1rem;
```

Comment:
The SCSS gets built on the server side, so I can't do any pre or post processing on the files without digging into the guts of it. I'm considering killing the fallback entirely, as it doesn't exist anywhere else I can see in the project.